x/configuration/client/cli/tx.go

```diff
@@ -40,6 +44,92 @@ var defaultDuration, _ = time.ParseDuration("1h")
 const defaultRegex = "^(.*?)?"
 const defaultNumber = 1
 
+func getCmdUpdateFee(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update-fee",
+		Short:   "update fee parameter(s) using key=value pair(s) noting that all numeric values should be specified in euros",
+		Example: "iovnscli tx configuration update-fee --from iovSAS --parameter fee_coin_price=0.29 --parameter register_domain_default=12",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc).WithBroadcastMode(flags.BroadcastBlock)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBuilder := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			// query the existing fee configuration
+			path := fmt.Sprintf("custom/%s/%s", types.StoreKey, types.QueryFees)
+			resp, _, err := cliCtx.Query(path)
+			if err != nil {
+				return err
+			}
+			var jsonResp types.QueryFeesResponse
+			err = queries.DefaultQueryDecode(resp, &jsonResp)
+			if err != nil {
+				return err
+			}
+			fees := jsonResp.Fees
+			// set new values via json tag https://gist.github.com/lelandbatey/a5c957b537bed39d1d6fb202c3b8de06
+			v := reflect.ValueOf(fees).Elem()
```

Comment:
Maybe using https://github.com/fatih/structs, would remove a lot of boilerplate code, other than that looks good to me

Reply:
Thanks for the suggestion.  I really want to specify parameters with json tags and, unfortunately, `structs` isn't helpful for that.